platform/ios/test/MGLMapViewConversionTests.m

```diff
@@ -0,0 +1,52 @@
+#import <Mapbox/Mapbox.h>
+#import <XCTest/XCTest.h>
+
+#import "../../ios/src/MGLMapView.h"
+
+@interface MGLMapViewConversionTests : XCTestCase
+
+@property (nonatomic) UIView *superView;
+@property (nonatomic) MGLMapView *mapView;
+
+@end
+
+@implementation MGLMapViewConversionTests
+
+- (void)setUp {
+    [super setUp];
+    
+    [MGLAccountManager setAccessToken:@"pk.feedcafedeadbeefbadebede"];
+    self.superView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, 64, 64)];
+    self.mapView = [[MGLMapView alloc] initWithFrame:CGRectMake(0, 0, 64, 64) styleURL:MGLStyle.streetsStyleURL];
```

Comment:
Unless this test depends on the Streets style specifically, replace this style URL with a local file URL pointing to one-liner.json, as in MGLStyleTests. That’ll avoid an unnecessary HTTP request that will error out.